EdurecaInClassProject/src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="footer">
+
+    <div class="container">
+      <div class="row mt-5 mb-5">
+        <div class="col-md-6 ">
+          <div class="card card-body box">
+            <h5 class="text-center mb-3">Popular Roads</h5>
+          
+                <ul class="row">
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Bangalore to Hubli</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Mumbai to Pune</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Chennai to Hubli</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Chennai to Bangalore</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Pune to Bangalore</li>
+                    <li class="col-md-6"><i class="fa fa-map-marker" aria-hidden="true" style="color:#F69230;font-size:17px;"></i>  Mumbai to Bangalore</li>
+                    
```

Comment:
Can we get this data from the component file? and use it using some directive.

Reply:
ngfor directive?

Reply:
yes